src/main/resources/js/main.js

```diff
@@ -12,6 +13,34 @@ function activeToggle() {
   });
 }
 
+var anchorForId = function (id) {
```

Comment:
It could be great if we could move these functions to Scala.js code, as it's a short script! Maybe for a future PR?